src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthLevelFour.java

```diff
@@ -13,7 +13,7 @@
  * @author Alexander Jesse
  * @see com.puppycrawl.tools.checkstyle.checks.coding.NestedForDepthCheck
  */
-public class InputNestedForDepth1 {
+public class InputNestedForDepthLevelFour {
```

Comment:
InputNestedForDepthLevelFour --> `InputNestedForDepthCheckCustomMaxLevelFour `

The word "Check" needs to be added by convention for all input files of src/test. Please read Issue description at #14436